kraken/kraken.json

```diff
@@ -2,119 +2,89 @@
-        "roles_key": "realm_access.roles",
-        "roles": ["admin"],
-        "roles_key_is_nested": true
-      }
-    },
-    "backend": [
+    {
+      "endpoint": "/api/v1/patients/{get1}/",
+      "input_headers":[
+          "*"
+      ],
+      "backend": [
         {
-          "host":["http://localhost:8080"],
-          "url_pattern": "/__health"
+          "host":["http://patients:8001"],
+          "url_pattern": "/{get1}",
+          "is_collection": true
         }
-    ]
-  },
-  {
-    "endpoint": "api/v1/validator",
-    "input_headers":[
-        "*"
-    ],
-    "extra_config": {
-      "auth/validator": {
-        "alg": "RS256",
-        "jwk_url": "http://keycloak:8080/realms/hms/protocol/openid-connect/certs",
-        "disable_jwk_security": true
-      }
+      ],
+      "method": "GET"
     },
-    "backend": [
-      {
-        "host":["http://validator:8000"],
-        "url_pattern": "/api/v1"
-      }
-    ]
-  },
-  {
-    "endpoint": "api/v1/patients",
-    "input_headers":[
+    {
+      "endpoint": "/api/v1/patients/{get1}/{get2}/",
+      "input_headers":[
         "*"
-    ],
-    "extra_config": {
-      "auth/patients": {
-        "alg": "RS256",
-        "jwk_url": "http://keycloak:8080/realms/hms/protocol/openid-connect/certs",
-        "disable_jwk_security": true
-      }
+      ],
+      "backend": [
+        {
+          "host":["http://patients:8001"],
+          "url_pattern": "/{get1}/{get2}",
+          "is_collection": true
+        }
+      ],
+      "method": "GET"
     },
-    "backend": [
-      {
-        "host":["http://patients:8001"],
-        "url_pattern": "/api/v1"
-      }
-    ]
-  },
-  {
-    "endpoint": "api/v1/resources",
-    "input_headers":[
+    {
+      "endpoint": "/api/v1/patients/{get1}/{get2}/{get3}/",
+      "input_headers":[
         "*"
-    ],
-    "extra_config": {
-      "auth/resources": {
-        "alg": "RS256",
-        "jwk_url": "http://keycloak:8080/realms/hms/protocol/openid-connect/certs",
-        "disable_jwk_security": true
-      }
+      ],
+      "backend": [
+        {
+          "host":["http://patients:8001"],
+          "url_pattern": "/{get1}/{get2}/{get3}",
+          "is_collection": true
+        }
+      ],
+      "method": "GET"
     },
-    "backend": [
-      {
-        "host":["http://resources:8002"],
-        "url_pattern": "/api/v1"
-      }
-    ]
-  },
-  {
-    "endpoint": "api/v1/documents",
-    "input_headers":[
-        "*"
-    ],
-    "extra_config": {
-      "auth/resources": {
-        "alg": "RS256",
-        "jwk_url": "http://keycloak:8080/realms/hms/protocol/openid-connect/certs",
-        "disable_jwk_security": true
-      }
+    {
+      "endpoint": "/api/v1/patients/{post1}/",
+      "input_headers":[
+          "*"
+      ],
+      "backend": [
+        {
+          "host":["http://patients:8001"],
+          "url_pattern": "/{post1}",
+          "is_collection": true
+        }
+      ],
+      "method": "POST"
```

Comment:
you think, maybe it is doubled, but unfortunately, for every method it should be a separate endpoint